frontend/src/v5/ui/routes/login/login.component.tsx

```diff
@@ -45,12 +48,13 @@ export const Login = () => {
 		},
 		resolver: yupResolver(LoginSchema),
 	});
-	const isPending = AuthHooksSelectors.selectIsPending();
 
+	const isPending = AuthHooksSelectors.selectIsPending();
 	const errorMessage: string = AuthHooksSelectors.selectLoginError();
 
 	const onSubmit = ({ username, password }) => {
 		AuthActionsDispatchers.login(username, password);
+		resetSSOParams();
```

Comment:
Instead of doing this I think is better to get reset the ssoParams on unmount of the component so we can tackle all other links and cases (like signup, forget password, etc)